docs/source/changelog.rst

```diff
@@ -20,6 +20,40 @@ We strongly recommend that you upgrade to version 9+ of pip before upgrading ``n
 
     Use ``pip install pip --upgrade`` to upgrade pip. Check pip version with
     ``pip --version``.
+    
+.. _release-5.6.0:
+
+5.6.0
+-----
+
+New features:
+
+- Execute cells by clicking area in input prompt (:ghpull:`3535`, :ghpull:`3687`)
+- Add kernel_info_timeout traitlet to enable restarting slow kernels (:ghpull:`3665`)
+- Add /metrics endpoint for Prometheus Metrics (:ghpull:`3490`)
+- Update to jQuery 3.3 (:ghpull:`3655`)
+
+Fixing problems:
+
+- Don't duplicate token in displayed URL (:ghpull:`3656`)
+- Don't trash non-empty directories on Windows (:ghpull:`3673`)
+- Include LICENSE file in wheels (:ghpull:`3671`)
+
+Testing: 
+- Add find replace test (:ghpull:`3630`)
+- [3335] Convert JS tests to Selenium (:ghpull:`3601`)
```

Comment:
Let's get rid of the `[3335]`, and preferably be more specific which JS tests were converted. Unfortunately PR titles don't all make good release notes.